day_7/fizzbuzz.rb

```diff
@@ -0,0 +1,34 @@
+numbers = [1]
+
+numbers.map {|n| numbers << n + 1
```

Comment:
How would using .each change the output of your code? Why use .map over .each in the case?